test/client/fixtures/runner/automation/select-element-test.js

```diff
@@ -591,139 +630,164 @@ $(document).ready(function () {
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
-    asyncTest('press right/left in select', function () {
-        var select = createSelect();
+            pressDownUpKeysActions(select);
+        });
 
-        $(select).attr('size', '2');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        asyncTest('press right/left in select', function () {
+            var select = createSelect();
 
-        pressRightLeftKeysActions(select);
-    });
+            $(select).attr('size', '2');
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
-    asyncTest('press down/up in select with "size" more than options count', function () {
-        var select = createSelect();
+            pressRightLeftKeysActions(select);
+        });
 
-        $(select).attr('size', '10');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        asyncTest('press down/up in select with "size" more than options count', function () {
+            var select = createSelect();
 
-        pressDownUpKeysActions(select);
-    });
+            $(select).attr('size', '10');
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
-    asyncTest('press right/left in select with "size" more than options count', function () {
-        var select = createSelect();
+            pressDownUpKeysActions(select);
+        });
 
-        $(select).attr('size', '10');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        asyncTest('press right/left in select with "size" more than options count', function () {
+            var select = createSelect();
 
-        pressRightLeftKeysActions(select);
-    });
+            $(select).attr('size', '10');
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
+            pressRightLeftKeysActions(select);
+        });
 
-    module('mouse actions with select element with attribute multiple');
-    asyncTest('click on option with scroll', function () {
-        var select = createSelect();
 
-        select.selectedIndex = 0;
-        equal(select.selectedIndex, 0);
-        $(select).attr('multiple', 'multiple');
-        clickPlaybackAutomation($(select).children(':last')[0], {}, function () {
-            equal(select.selectedIndex, 4);
-            startNext();
+        module('mouse actions with select element with attribute multiple');
```

Comment:
module('mouse actions with the "select" element with the "multiple" attribute');